blockly-rtc/src/websocket_handlers.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * 
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Realtime collaboration demo using Node.js and SQlite.
+ * @author navil@google.com (Navil Perez)
+ */
+
+import io from 'socket.io-client';
```

Comment:
Does ``socket.io-client`` need to be added to package.json?

Reply:
or is it included in the ``socket.io`` package?

Reply:
Discussed offline. It is imported in the socket.io package, but we are still including it in package.json to convey that it does need to be imported/